requirements.test.txt

```diff
@@ -0,0 +1,3 @@
+pytest-asyncio-0.18.3
+pytest==7.1.1
+pytest-cov==3.0.0
```

Comment:
Locally I've been using `pytest-cov` to generate a coverage report when executing the tests:
```
pytest --cov=sharkiq --cov-report html --cov-report term
```

Reply:
Workflow has been added 👍